keyboards/al1/matrix.c

```diff
@@ -28,6 +28,13 @@ inline uint8_t matrix_cols(void) {
   return MATRIX_COLS;
 }
 
+void matrix_init_kb(void) {
```

Comment:
For consistency, these should be set as "weak".

Reply:
Done